docs/dev_guide/biodata/buddies/inject.rst

```diff
@@ -0,0 +1,100 @@
+Injecting the Buddy Service
+=============================
+
+A typical use case for a Chado Buddy is in an importer.
+Best practice is to inject the buddy service into your importer class.
+Let's see how that is done.
+
+First you will need to include these classes in your importer:
+
+.. code::
+
+  use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+  use Symfony\Component\DependencyInjection\ContainerInterface;
+  use Drupal\tripal_chado\ChadoBuddy\PluginManagers\ChadoBuddyPluginManager;
+
+Then you will need to implement ``ContainerFactoryPluginInterface`` in your class. For example:
+
+.. code::
+
+  abstract class MyImporter extends ChadoImporterBase implements ContainerFactoryPluginInterface {
```

Comment:
Don't suggest abstract in the example since it is more likely they will want a fully fledged class here.

```suggestion
  class MyImporter extends ChadoImporterBase implements ContainerFactoryPluginInterface {
```